crates/sui-rpc-api/proto/sui.types.proto

```diff
@@ -0,0 +1,1040 @@
+syntax = "proto3";
+
+package sui.types;
+
+import "google/protobuf/empty.proto";
+import "google/protobuf/timestamp.proto";
+
+message Address {
+  bytes address = 1;
+}
+
+message ObjectId {
+  bytes object_id = 1;
+}
+
+message Digest {
+  bytes digest = 1;
+}
+
+message Bcs {
+  bytes bcs = 1;
+  // optional bytes name = 2;
+}
+
+// Checkpoint {{{
+
+message CheckpointSummary {
+  // Reserve field 1 and 2 for possible bcs bytes and/or Digest of type
+  reserved 1, 2;
```

Comment:
if you use the actual "reserved" keyword it generally indicates these tag numbers should never be used in the future

if you just want to save them for future use, I'd keep the comment and remove the "reserved"

(same below)

Reply:
oh i didn't realize that, easy enough to make this change